code/datums/spells/demon_charge.dm

```diff
@@ -0,0 +1,93 @@
+/obj/effect/immovablerod/demon
+	name = "Demon"
+	desc = "Oh no!"
+	icon = 'icons/mob/mob.dmi'
+	icon_state = "daemon"
+	var/mob/living/caster
+	var/max_distance = 7
+	var/turf/start_turf
+	notify = FALSE
+
+/obj/effect/proc_holder/spell/demon_charge
+	name = "Demon Charge"
+	desc = "Take on the form of an immovable rod, destroying all in your path."
```

Comment:
im guessing this was a copy paste from the wizard spell, but this will probably need a more.. bloody theme